gems/pending/appliance_console/external_httpd_authentication.rb

```diff
@@ -108,6 +108,7 @@ def configure_ipa
       say("\nConfiguring IPA (may take a minute) ...")
       ipa_client_unconfigure if ipa_client_configured?
       ipa_client_configure(realm, @domain, @ipaserver, @principal, @password)
+      enable_kerberos_dns_lookups
```

Comment:
what about when IPA client is unconfigured ? what happens to the /etc/krb5.conf file, is it copied back by IPA to the original or do we need to update those parameters back to "false" ?